src/App.js

```diff
@@ -1,25 +1,111 @@
-import logo from './logo.svg';
-import './App.css';
-
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Initial Commit
-        </a>
-      </header>
-    </div>
-  );
+import React, { useEffect, useState } from 'react';
+import { questions } from './data/Questions.js';
+import { fetchData } from './apiCalls';
+import { Route, Link } from 'react-router-dom';
+import Results from './Results';
+import HomeQuiz from './HomeQuiz';
+import { getCharacterInfo } from './util';
+// import Error from './Error';
+
+
+
+export default function App() {
+	const [currentQuestion, setCurrentQuestion] = useState(0);
+
+	// Total score per character setters
+	const [totalMorty, setMorty] = useState(0);
+	const [totalRick, setRick] = useState(0);
+	const [totalJerry, setJerry] = useState(0);
+	const [totalSummer, setSummer] = useState(0);
+
+	//Final result setter
+	const [character, setCharacter] = useState("Beth");
+
+	//const [showQuiz, setShowQuiz] = useState(false);
+	const [showCharacter, setShowCharacter] = useState(false);
+  const [banner, setBanner] = useState('')
+  const [characterInfo, setCharacterInfo] = useState([])
+
+  useEffect(() => {
```

Comment:
I think fetches aren't suppose to happen inside of useEffect. You can use a getData () to call the fetch and call the function in use effect.